app/build.gradle

```diff
@@ -28,6 +28,8 @@ dependencies {
     api 'androidx.appcompat:appcompat:1.0.0'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation project(path: ':truffleshuffle')
+    implementation "androidx.cardview:cardview:1.0.0"
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
```

Comment:
Is the recycler view dependency needed?

Reply:
I tested the changes without the recycler view and it works as expected.